### ISSUE TITLE
AJAX calls with errors should not render HTML

### ISSUE THREAD

**Issue by ericclemmons** [ericclemmons/polydev#39]
The error handler should be skipped when the request is AJAX.

**Comment by willbuyers412** [comment]
I completely agree that AJAX calls with errors should not render HTML, especially when it comes to a critical task like [GlenAllen Replacement](https://garagedoorrepairglenallen.net/garage-door-replacement/). This can cause confusion and potentially lead to data loss or other issues. It's important to handle errors gracefully and provide clear feedback to the user.
